src/signal.rs

```diff
@@ -18,9 +18,9 @@ use gdk::{
 use cairo::{Context, RectangleInt};
 
 use {
-    Adjustment, Button, Dialog, DirectionType, Range, ScrollType, SpinButton, StateFlags,
-    TextDirection, ToolButton, Tooltip, TreeIter, TreePath, TreeSelection, TreeView,
-    TreeViewColumn, Widget, WidgetHelpType,
+    Adjustment, Button, DeleteType, Dialog, DirectionType, Entry, MovementStep,
```

Comment:
Should I put each of these on their own lines to avoid this much churn in the future?